wirepas_mesh_messaging/event.py

```diff
@@ -18,17 +18,21 @@ class Event(object):
         gw_id (str): gateway unique identifier
         sink_id(str): sink identifier
         event_id(int): event unique id (random value generated if None)
+        time_ms_epoch(int): timestamp in ms of event generation (0 to automatically set it, the default)
     """
 
     # pylint: disable=unused-argument
-    def __init__(self, gw_id, sink_id=None, event_id=None, **kwargs):
+    def __init__(self, gw_id, sink_id=None, event_id=None, time_ms_epoch=0, **kwargs):
 
         super(Event, self).__init__()
         self.gw_id = gw_id
         self.sink_id = sink_id
         if event_id is None:
             event_id = random.getrandbits(64)
         self.event_id = event_id
+        if time_ms_epoch == 0:
+            time_ms_epoch = int(time.time() * 1000)
+        self.time_ms_epoch = time_ms_epoch
```

Comment:
you can do this as a oneliner: 
self.time_ms_epoch = time_ms_epoch if time_ms_epoch!=0 else int(time.time() * 1000)

Reply:
Better to keep as above.
And to be honest, I prefer the multi-line approach :-)